src/main/java/org/example/Contact.java

```diff
@@ -0,0 +1,35 @@
+package org.example;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Contact {
+
+
+    public String name;
+    public String phoneNumber;
+
+    public Contact(String name, String phoneNumber){
+        this.name=name;
+
+        if(phoneNumber.length()!=10){
+            this.phoneNumber=null;
```

Comment:
throw an exception here, if you leave the code like this, the phone will save empty numbers and won't warn the user

Reply:
and congrats on adding the validation